### PR TITLE
Add export_tracing_func to _GlobalSw

### PR DESCRIPTION
**Pull request by rjain11**
`_GlobalSw` doesn't allow to set a `export_tracing_func` that contains the span annotations. Add a param to expose this functionality.

### CODE REVIEW

**Comment by nipunn1313** [comment]
Can you also update test_stopwatch_global.py?

**Comment by rjain11** [comment]
I've added a pretty basic check to test the new path. Let me know if we need something more comprehensive here.

File: test_stopwatch_global.py [hunk 1]
```diff
@@ -4,20 +4,38 @@
...
+        with global_sw().timer('parent', start_time=20, end_time=80):
+            global_sw().add_span_annotation('parent_annotation', 1)
+            with global_sw().timer('child', start_time=40, end_time=40):
+                global_sw().add_span_annotation('child_annotation', 1)
+
+    def test_reported_traces(self):
+        tracing_function = Mock()
+        global_sw_init(export_tracing_func=tracing_function)
+        self.add_spans()
+        reported_traces = global_sw().get_last_trace_report()
```
Comment (nipunn1313) [inline test_stopwatch_global.py]: I think it's important to verify that reported_traces includes 'parent_annotation' and 'child_annotation' as expected.

This test would pass if reported_traces is empty, so it doesn't get to the crux of what we are trying to defend against with our testing.

Comment (rjain11) [inline test_stopwatch_global.py]: Makes sense, adding a couple of more asserts to test this.

**Pull request merged**